addons/binding/org.openhab.binding.zway/README.md

```diff
@@ -153,6 +155,10 @@ Because textual configuration isn't useful, follow the instructions in the [Gett
 
 ## Developer stuff
 
+### Known issues
+
+- The Z-Way Binding only works, when simple mode of item linking is enabled during thing creation.
```

Comment:
I have created a topic in openHAB forum: https://community.openhab.org/t/things-with-multiple-channels-of-the-same-channel-type/21118?u=pathec.